package.json

```diff
@@ -60,9 +61,15 @@
 		"node": "14.x || >= 16"
 	},
 	"peerDependencies": {
-		"eslint": ">=8.0.0"
+		"eslint": ">=8.0.0",
+		"vitest": "*"
```

Comment:
Would it be possible to mark `vitest` as optional also?  (Similar to how `eslint-plugin-jest` marks `jest` as optional)